src/main/resources/email-templates/certificate-returned-la-emails.html

```diff
@@ -0,0 +1,10 @@
+<html>
+  <body>
+    <p>The print provider has notified us that a postal certificate has been returned.</p>
+    <p>You can access the application here:</p>
+    <p><a href="${applicationUrl}">${applicationUrl}</a></p>
+    <br>
+    <p>Local Authority email addresses: ${localAuthorityEmailAddresses}</p>
```

Comment:
Is it worth stating somewhere (perhaps at the top) in this email content that `NOTE: This is a Sample/Test email` ?